packages/devextreme/js/__internal/grids/grid_core/columns_resizing_reordering/m_columns_resizing_reordering.ts

```diff
@@ -49,19 +50,23 @@ const allowReordering = function (that) {
   return that.option('allowColumnReordering') || that.getController('columns').isColumnOptionUsed('allowReordering');
 };
 
-const TrackerView = modules.View.inherit({
+class TrackerView extends modules.View {
+  private _positionChanged: any;
```

Comment:
Let's declare the ```_positionChanged``` as a private method.
Because the method implementation isn't changed in runtime.
And add/remove this method to ```_tablePositionController``` on subscribe/unsubscribe